packages/storybook/stories/components/radio/documentation.mdx

```diff
@@ -1,65 +1,123 @@
+          style={ { fontSize: '.7rem', paddingLeft: '.3rem', paddingRight: '.2rem' } }></OdsIcon></a><br />
+        <a href='https://github.com/ovh/design-system/tree/master/packages/components/src/radio'>Github<OdsIcon
+          name={ ODS_ICON_NAME.externalLink }
+          style={ { fontSize: '.7rem', paddingLeft: '.3rem', paddingRight: '.2rem' } }></OdsIcon></a>
+      </td>
+    </tr>
+    </tbody>
+  </table>
+</div>
+
+## Anatomy
+
+![Component anatomy](components/radio/anatomy.png "Component anatomy")
+
+1. **Radio** : The selection control
+
+## Usage
+
+In most situations where you want to present a list of mutually exclusive options, **you will want to group Radio buttons**.
+
+**Radio** groups can be used within a form.
+
+Also, it can serve as :
+- Selecting choice
+- Lists/sub-lists
+- Filters
+
+### Dos & Don'ts
+
+<Markdown>
+  { `
+| ✅ Do                                                                                                                                                                                   |
+|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------         |
+| - Always use clear and concise labels for **Radio** buttons.                                                                                                                           |
+| - Long labels may wrap to a second line, and this is preferable to truncation.                                                                                                         |
+| - If using two affirmative/negative-type **Radio** buttons, you might consider converting it to a sole [**Checkbox**](?path=/docs/ods-components-form-checkbox--documentation) button. |
+` }
+</Markdown>
+<Markdown>
+  { `
+| ❌ Don't                                                                                                                                                                          |
+|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| - Use **Radio** buttons when multiple items can be selected, use [**Checkbox**](?path=/docs/ods-components-form-checkbox--documentation) buttons instead.                        |
+| - Use more than 5 **Radio** buttons, instead convert it to a [**Select**](?path=/docs/ods-components-form-select--documentation).                                                |
+| - Avoid situations where all options in a set should be deselected.                                                                                                              |
+` }
+</Markdown>
+
+## Placement
+
+**Radio** groups can be either horizontal or vertical.
+
+By default, **Radio** groups are vertical, meaning one under another.
+
+Use a horizontal **Radio** group only when vertical space is limited.
```

Comment:
Usage of horizontal radio groups should only occur if vertical space is limited.